doc/conf.py

```diff
@@ -30,7 +30,7 @@
 
```

Comment:
Can you remove the now unused "import datetime" above?

Reply:
yaa sure